### PR TITLE
feat: create typed-res as a devtool

### PR DESCRIPTION
**Pull request by egdiala**

### CODE REVIEW

File: src/background.ts [hunk 1]
```diff
@@ -5,3 +5,8 @@ console.log("Hello from the background!");
 browser.runtime.onInstalled.addListener((details) => {
   console.log("Extension installed:", details);
```
Comment (Nwosu-Promise) [inline src/background.ts:6]: Please, remove this console.log here.

File: src/pages/Popup.vue [hunk 2]
```diff
@@ -1,13 +1,24 @@
 <script lang="ts" setup>
+import Browser from 'webextension-polyfill';
+
 console.log("Hello from the popup!");
```
Comment (Nwosu-Promise) [inline src/pages/Popup.vue:4]: Please, remove this console.log here

File: src/pages/Popup.vue [hunk 3]
```diff
@@ -1,13 +1,24 @@
...
+
+Browser.runtime.onInstalled.addListener((details) => {
+  console.log("Extension installed:", details);
```
Comment (Nwosu-Promise) [inline src/pages/Popup.vue:7]: Please, remove this console.log here

**Comment by egdiala** [comment]
Won't merge as dev branch is already up to date

**Pull request closed without merging**